demos/1p_caiman/actors/1p_processor.py

```diff
@@ -6,8 +6,8 @@
 from caiman.source_extraction.cnmf.online_cnmf import OnACID
```

Comment:
Same idea here: do a `try`, `except` block telling users that CaImAn is required for the demo. @draelos, we had discussed having a `caiman` optional install (e.g., `pip install improv[caiman]`). Should we do that here? Or just instruct users to do it when they try to run something like this?

Reply:
I think it's better for now to just tell people to `pip install caiman` rather than building our own optional dependency. 

The only complexity to be aware of here is making sure everything shuts down cleanly after the error.

Reply:
We could write something into Nexus where it loads the actors via Tweak (renamed Config yet?), catches the error, and then Nexus can itself take care of shutdown per usual. 
This would also start to help resolve issues where 1 actor has a code error on loading and everything else is fine -- this can be missed since the error is only in the logfile and everything else is smooth. Probably should discuss this further in the next meeting.